src/legacy/ui/public/courier/courier.js

```diff
@@ -62,29 +57,12 @@ uiModules.get('kibana/courier').service('courier', ($rootScope, Private) => {
         // resume polling.
         searchPoll.pause();
 
-        // And abort all pending requests.
-        searchRequestQueue.abortAll();
-
-        if (searchRequestQueue.getCount()) {
-          throw new Error('Aborting all pending requests failed.');
-        }
-
         refreshIntervalSubscription.unsubscribe();
       });
 
       addFatalErrorCallback(closeOnFatal);
       updateRefreshInterval();
     }
-
-    /**
-     * Fetch the pending requests.
-     */
-    fetch() {
```

Comment:
This method was only used when consumers of `SearchSource` used `searchSource.onResults()`, all of which have now been replaced with `searchSource.fetch()`